fabric_rti_mcp/services/map/map_tools.py

```diff
@@ -23,6 +23,41 @@ def register_tools(mcp: FastMCP) -> None:
         annotations=ToolAnnotations(readOnlyHint=True, destructiveHint=False),
     )
 
+    mcp.add_tool(
```

Comment:
can we not create so many verbose tools? create one tool for getting schema, and add a variable for type, or whatever this is. generally speaking, this server is not meant to be control plane, fabric mcp is there for that, so make sure you provide value beyond crud.